calico-cloud/_includes/components/ReqsSys.js

```diff
@@ -359,6 +359,13 @@ function NetworkRequirementsEnt(props) {
             <td>Additional port required for Manager UI</td>
             <td>TCP 9449</td>
           </tr>
+          <tr className='odd'>
+            <td>
+              <strong>Egress gateway</strong>
+            </td>
+            <td>{prodname} compliance</td>
```

Comment:
I think this needs to be about egw. A few other instances below

Reply:
Line 366

Reply:
Fixed.